### ISSUE TITLE
Converting assets (/graphics) on windows, strange freeze/crash

### ISSUE THREAD

**Issue by TakesTheBiscuit** [SFTtech/openage#1091]
tldr; multi thread SLP conversion on my box was crashing at the same place every time - disabling that code has worked...

Hi there

Windows 10 pro; i7 processor 16GB ram..

I've had a fair bit of pain compiling/cmaking and now i'm onto conversion - not without it's challenges (my version was too new so i've opted in to the beta version in steam).

Running conversion crashed every time on `graphics/1019.slp`
Deleted the file, but then it would crash on 102.. deleted that.. same..
So it seems that after all audio was converted and about 6 graphic conversions it was just 'sitting there' - powershell just had a blinking cursor and it wouldn't move to the next file.

I've been digging about and changed `slp_converter_pool.py`  to see if i could get any reason or debug;
Modified the free memory check to:
``` python
if True:
    # TODO print the warn only once
    warn("Low on memory; disabling parallel SLP conversion")
```

Ok so now it's taking a long time to convert graphics but at least they are being converted.

I do have another conversion error or run error to do with fonts but i'll see what's going on there.

Not sure why disabling the multi-thread code helped me here but it did.

**Comment by TheJJ** [comment]
As it's hard or impossible for me to reproduce this, I'll need your assistance in tracking down the problem with multiprocessing. I'm wondering though what the difference is because many people have converted the assets to date an nobody had crashing problems due to multiprocessing (at least I've never heard that).

So, how did it crash on `1019.slp`? Any traceback? :smiley:

**Comment by TakesTheBiscuit** [comment]
I can definitely see what output i can get - i guess i would need a way for the threads to report output somewhere - maybe a log file? Will have another look soon!

**Comment by simonsan** [comment]
Seems like we have two new cases here:
https://www.reddit.com/r/openage/comments/c1bcqs/installer_for_windows_10_64_bit_is_now_available/erdw0r0/

**Comment by ms111ds** [comment]
I'm running into the same problem. 1019.slp is the eighth slp file to be processed and it looks like the blocking is happening at this point because I have 8 cpus in my system. So an .slp conversion starts on each cpu and then they all block.

I think the reason this is happening is because `converter_process()` in slp_converter_pool.py is not starting and stuffing the output queue. So when `convert()` is called, it blocks on the line `result = outqueue.get()` since the output queue is empty. I can't figure out why `converter_process()` doesn't start but the only clue is the error below when `process.start()` is called SLPConverterPool's initialization. 

<img width="629" alt="converter_process_error" src="https://user-images.githubusercontent.com/38050737/61198434-03cd1800-a6a8-11e9-8a0a-74d187a5bafb.png">

**Comment by heinezen** [comment]
This specific bug cannot occur anymore as we removed multiprocessing from this part of the conversion pipeline.
